.github/workflows/deploy.yml

```diff
@@ -51,13 +51,44 @@ jobs:
           echo "Tag will not be created."
         fi
 
+  plan_builds:
+    name: Plan builds
+    runs-on: ubuntu-latest
+    outputs:
+      download_linux_arm32_runid: ${{ steps.plan_builds.outputs.download_linux_arm32_runid }}
+      download_linux_arm64_runid: ${{ steps.plan_builds.outputs.download_linux_arm64_runid }}
+      download_linux_x64_runid: ${{ steps.plan_builds.outputs.download_linux_x64_runid }}
+      download_macos_runid: ${{ steps.plan_builds.outputs.download_macos_runid }}
+      download_windows_x64_runid: ${{ steps.plan_builds.outputs.download_windows_x64_runid }}
+      download_windows_portable_runid: ${{ steps.plan_builds.outputs.download_windows_portable_runid }}
+      platforms_to_build: ${{ steps.plan_builds.outputs.platforms_to_build }}
```

Comment:
Maybe:

```YAML
reuse_runs:
  outputs:
    linux_arm32: ...
    linux_arm64: ...
    ...
    except_these:
```

Or even:

```YAML
reuse_runs:
  outputs:
    except_these: ...
  steps:
  - Plan builds
  - Download artifacts from other runs
  - Upload as artifacts to this run
```

Reply:
Another possibility is to combine the planning job with the `get_tag_name` job:

```YAML
plan:
  outputs:
    tag_name: ${{ steps.make_tag_name.outputs.tag_name  }}
    release_name:  ${{ steps.make_tag_name.outputs.release_name }}
    ...
    # maybe, or re-upload as above
    reuse_run_linux_arm32: ${{ steps.plan_builds.outputs.reuse_run_linux_arm32 }}
    reuse_run_linux_arm64: ${{ steps.plan_builds.outputs.reuse_run_linux_arm64 }}
    ...
    build_platforms: ${{ steps.plan_builds.outputs.build_platforms }}
  steps:
  - id: make_tag_name
  - id: plan_builds
 ```
Advantage: fewer jobs in the [summary graph](https://github.com/musescore/MuseScore/actions/runs/10507169262).